blurb/tests/test_blurb.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+from pyfakefs.fake_filesystem import FakeFilesystem
+
+import blurb
+
+
+@pytest.mark.parametrize(
+    "section, expected",
+    (
+        ("C API", "C API"),
+        ("Core and Builtins", "Core and Builtins"),
+        ("Library", "Library"),
+        ("Tools/Demos", "Tools-Demos"),
+    ),
+)
+def test_sanitize_section(section: str, expected: str) -> None:
+    sanitized = blurb.sanitize_section(section)
+    assert sanitized == expected
+
+
+@pytest.mark.parametrize(
+    "section, expected",
+    (
+        ("C API", "C API"),
+        ("Core and Builtins", "Core and Builtins"),
+        ("Library", "Library"),
+        ("Tools-Demos", "Tools/Demos"),
+    ),
+)
```

Comment:
Instead of (almost) duplicating this, what about creating a list of idempotent names and a mapping (or two) of names that get transformed and then test those separately?

I'm also curious to see a few more examples (e.g. `"C api"`, `"Tools\Demos"`, etc.), but those could be added in a separate PR.

Reply:
This still doesn't solve the duplication between `test_unsanitize_section_no_change` and `test_sanitize_section_no_change` though.

Reply:
Updated! https://github.com/python/core-workflow/pull/520/commits/70f600460e435f4c365d2b9d7a91cf527974f82b